### ISSUE TITLE
Was It Installed???

### ISSUE THREAD

**Issue by Animationss** [L8X/Roblox-Client-Optimizer#6]
Got a bit of a predicament, So I ran both the .bat and .cmd files with and without roblox open and running but im not sure if the script

 didn't work, hasn't finished doing its thing, or if It alr did its thing.  

If it did install/inject can you make a way to identify that, Maybe a loading screen or something.

![image](https://user-images.githubusercontent.com/89710296/204163199-82f6133d-fd15-472e-917d-71e9bcd4c18b.png)

**Comment by Kaiddd** [comment]
@Animationss did you disable UAC prompts on your windows or something? You should get a pop-up asking for admin

**Comment by Animationss** [comment]
No, But I am using a custom windows 10 iso (GGOS) and I wouldn't be surprised if it was turned off by default.

**Comment by Kaiddd** [comment]
@Animationss thats not where you configure UAC, there's a menu with a slider, I'm at work rn so you're gonna have to find it yourself, put it on the highest or second highest mode, believe second highest is default? Correct me if wrong.

**Comment by Kaiddd** [comment]
If you'd like to contact me on discord for faster response times add me at Kaid#0001

**Comment by Kaiddd** [comment]
Also, if you end up not getting your UAC to work due to your custom iso, try running the batch script as administrator, very well may just work if you do that, not sure though as it does try to prompt UAC regardless I believe? Not sure, the admin batch UAC prompt part of the installer is the only part I didn't make, that's from stackoverflow, invoking an admin shell myself is past my batch knowledge haha, could easily study the current one and recode it if you can't figure out installing it even with my help though.

**Comment by Kaiddd** [comment]
Fixed in #15
